lesson_26/script.js

```diff
@@ -0,0 +1,67 @@
+
+function Accumulator(initialValue) {
+    this.value = initialValue;
+    this.increment = function() {
+        this.value++;
+    };
+    this.decrement = function() {
+        this.value--;
+    };
+}
+
+function CancelableAccumulator(initialValue) {
```

Comment:
you also had to change to prototype of this function to make inheritance

see the presentation to this lesson